ng-intl-tel-input.directive.js

```diff
@@ -14,6 +14,10 @@ angular.module('ngIntlTelInput')
           if (attr.defaultCountry) {
             ngIntlTelInput.set({defaultCountry: attr.defaultCountry});
           }
+          // Override default number type.
+          if (attr.defaultNumberType) {
```

Comment:
I feel like this should check that the proper `FIXED_LINE` or `MOBILE` values are being assigned, but the underlying library may help cover that base.